src/ImageSharp/Formats/Gif/GifColorTableMode.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Formats.Gif
+{
+    /// <summary>
+    /// Provides enumeration for the available Gif color table modes.
+    /// </summary>
+    public enum GifColorTableMode
+    {
+        /// <summary>
+        /// A single color table is calculated from the first frame and reused for subsequent frames.
```

Comment:
We should probably allow the user to select the baseline frame.

Reply:
I did think about this but if someone creates a custom `PaletteQuantizer` implementation then it doesn't matter what frame is used. Plus it would vary from image to image.